src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -337,9 +338,13 @@ protected function describeContainerDefinition(Definition $definition, array $op
                     $argument = $argument->getValues()[0];
                 }
                 if ($argument instanceof Reference) {
-                    $argumentsInformation[] = sprintf('Service(%s)', (string) $argument);
+                    $argumentsInformation[] = sprintf('Service (%s)', (string) $argument);
```

Comment:
this is unrelated and should be reverted - it can break scripts that parse the output

Reply:
Hopefully, such script should use the json or XML output, but yes, I still vote for reverting.

Reply:
reverted